src/layers/pool.jl

```diff
@@ -143,3 +143,35 @@ function topk_index(y::AbstractVector, k::Int)
 end
 
 topk_index(y::Adjoint, k::Int) = topk_index(y', k)
+
+"""
+    WeigthAndSumPool(in_feats)
+
+WeigthAndSum sum pooling layer.
+Takes a graph and the node features as inputs, computes the weights for each node and perform a weighted sum.
+"""
+struct WeigthAndSumPool
+    in_feats::Int
+    dense_layer::Dense
+end
+
+@functor WeigthAndSumPool
+
+Flux.trainable(ws::WeigthAndSumPool) = (ws.dense_layer)
```

Comment:
```suggestion
Flux.trainable(ws::WeigthAndSumPool) = (ws.dense_layer,)
```
this should return a tuple. 
But actually you can remove this entirely, it is implicitly defined by `@functor`